File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/cpp/cwe-89/code_before3.cpp
// From cwe-snippets, snippets_2/non-compliant/C++/0057.cpp

sprintf (sql, "SELECT * FROM items WHERE owner='%s' AND itemname='%s'", username, request.Lookup("item"));
printf("SQL to execute is: \n\t\t %s\n", sql);
rc = sqlite3_exec(db,sql, NULL,0, ?err);

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/python/cwe-89/code_before3.py
# From cwe-snippets, snippets_2/non-compliant/Python/0059.py

userName = req.field('userName')
itemName = req.field('itemName')
query = "SELECT * FROM items WHERE owner = ' " + userName +" ' AND itemname = ' " + itemName +"';"
cursor.execute(query)
result = cursor.fetchall()